multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1470,6 +1479,65 @@ def _on_predict_start(
 
         return data, df_preprocessor, data_processors
 
+    def evaluate_coco(
```

Comment:
Maybe we could make this a util function and call it in the `evaluate()` API. We can detect if `self._pipeline == OBJECT_DETECTION`.

Reply:
Done.